RecoEgamma/EgammaIsolationAlgos/python/eleIsoDeposits_cff.py

```diff
@@ -6,11 +6,11 @@
 from RecoEgamma.EgammaIsolationAlgos.eleIsoDepositEcalFromHits_cff import *
 from RecoEgamma.EgammaIsolationAlgos.eleIsoDepositHcalFromTowers_cff import *
 
-eleIsoDeposits = cms.Sequence(
-    eleIsoDepositTk +
-    eleIsoDepositEcalFromHits + 
-    eleIsoDepositHcalFromTowers +
-    eleIsoDepositHcalDepth1FromTowers +
+eleIsoDepositsTask = cms.Task(
+    eleIsoDepositTk ,
+    eleIsoDepositEcalFromHits , 
+    eleIsoDepositHcalFromTowers ,
+    eleIsoDepositHcalDepth1FromTowers ,
     eleIsoDepositHcalDepth2FromTowers
 )
-
+eleIsoDeposits = cms.Sequence(eleIsoDepositsTask)
```

Comment:
This is only used in ```RecoEgamma/EgammaIsolationAlgos/python/eleIsolationSequence_cff.py```: maybe one could avoid defining explicitly the sequence here